mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -819,6 +828,8 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	METHOD m_yndwkmtt animateDamage ()V
 	METHOD m_ynhqjslw getX ()D
 	METHOD m_yoauovfn playExtinguishSound ()V
+	METHOD m_yontuqgh closerThan (Lnet/minecraft/unmapped/C_astfners;DD)Z
```

Comment:
```suggestion
	METHOD m_yontuqgh isCloserThan (Lnet/minecraft/unmapped/C_astfners;DD)Z
```

Reply:
> Boolean field and variable names should always be adjective phrases or present tense verb phrases (`powered`, `canOpen`),
avoiding the `is` and `has` prefixes when possible (`colored`, not `isColored` or `hasColor`).

Reply:
agree with Matrix, this format again has been used to describe boolean methods for other classes, therefore it MUST be followed to prevent any confusion.

Reply:
also, what is it closer to? I havent looked at the warden class as much since im studying, but what exactly would this be looking for? Is closer than what?

Reply:
> > Boolean field and variable names should always be adjective phrases or present tense verb phrases (`powered`, `canOpen`),
> avoiding the `is` and `has` prefixes when possible (`colored`, not `isColored` or `hasColor`).

> Boolean fields


This is a method...

Reply:
I meant if it returns a boolean

Reply:
I am well aware that it is a method

Reply:
> > > Boolean field and variable names should always be adjective phrases or present tense verb phrases (`powered`, `canOpen`),
> > > avoiding the `is` and `has` prefixes when possible (`colored`, not `isColored` or `hasColor`).
> 
> > Boolean fields
> 
> This is a method...

oh yes nvm then

Reply:
> also, what is it closer to? I havent looked at the warden class as much since im studying, but what exactly would this be looking for? Is closer than what?

first param is entity